<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetHandFeatureClass</class>
 <widget class="QWidget" name="GetHandFeatureClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GetHandFeature</string>
  </property>
  <widget class="QLabel" name="DepthLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{border:3px solid #242424;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="IRLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{border:3px solid #242424;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>参数设置</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>滑动步长：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TxtSliplen">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'楷体'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;15&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>窗口宽度：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TxtSlipwidth">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'楷体'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;30&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Txtwait">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'楷体'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;10&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>wait：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rate：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Txtrate">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'楷体'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0.4&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>读取数据集</string>
   </property>
   <widget class="QToolButton" name="btnPath">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>18</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Adobe Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Path：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TXpath">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>有效数据集：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TXefficient">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>381</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="BtnStart">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;border:2px groove gray;</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>异常数据样本：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Abnoraml_Data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnDel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;border:2px groove gray;</string>
    </property>
    <property name="text">
     <string>删除异常样本</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GetHandFeature.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BtnStart</sender>
   <signal>clicked()</signal>
   <receiver>GetHandFeatureClass</receiver>
   <slot>hand_detect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPath</sender>
   <signal>clicked()</signal>
   <receiver>GetHandFeatureClass</receiver>
   <slot>btn_ChooseData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>792</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>832</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnDel</sender>
   <signal>clicked()</signal>
   <receiver>GetHandFeatureClass</receiver>
   <slot>hand_sample_del()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>658</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>819</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>hand_detect_clicked()</slot>
  <slot>btn_ChooseData()</slot>
  <slot>btn_Save()</slot>
  <slot>hand_sample_del()</slot>
 </slots>
</ui>
